File: 70-climbing-stairs/climbing-stairs.py
class Solution:
    def climbStairs(self, n):

        if n <= 2:
            return n

        first = 1
        second = 2
        i = 3

        while i <= n:

            current = first + second

            first = second
            second = current

            i += 1

        return second

File: 70-climbing-stairs/climbing-stairs.java
class Solution {
    public int climbStairs(int n) {
        if(n<=3)
          return n;
        int f=1;
        int s=2;
        int i=3;
        while(i<=n){
            int c=f+s;
            f=s;
            s=c;
            i++;
        }
        return s;
    }
}